tests/acceptance/commercial/test_monitor_addon.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import subprocess
+
+import pytest
+
+from utils.common import (
+    build_image,
+    latest_build_artifact,
+)
+
+@pytest.mark.commercial
+@pytest.mark.min_mender_version("3.1.0")
+class TestDeltaUpdateModule:
+    @pytest.mark.only_with_image("ext4")
+    def test_build_addon(
+        self, request, bitbake_variables, prepared_test_build, bitbake_image
+    ):
+
```

Comment:
let me guess, black complains about that blank line?

Reply:
:+1: